superset/views/base.py

```diff
@@ -155,6 +155,26 @@ def wraps(self, *args, **kwargs):
     return functools.update_wrapper(wraps, f)
 
 
+def api_exists_owned(f):
```

Comment:
Not wanting to get hung up on details, but personally I would prefer something slightly more verbose, e.g. `check_ownership_and_item_exists`.